src/software/geom/shot.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "software/geom/point.h"
+
+class Shot
+{
+   public:
+    /**
+     * Creates a shot with the given target and angle
+     *
+     * @param point The target to shoot at
+     * @param angle The angle between the obstacles on either side of the shot vector
+     *
+     */
+    Shot(Point point, Angle angle);
+
+    /**
+     * Returns a point, the target to shoot at.
+     * @return a point, representing the target to shoot at.
+     */
+    Point getPoint() const;
+
+    /**
+     * Returns an angle between the obstacles on either side of the shot vector.
+     * @return an angle between the obstacles on either side of the shot vector.
+     */
+    Angle getAngle() const;
```

Comment:
Perhaps rename to `getOpenAngle()`? Again, just being really explicit so that someone doesn't incorrectly use the value from this.